### PR TITLE
Homepage v2

### PR DESCRIPTION
**Pull request by hawkticehurst**

### CODE REVIEW

File: src/components/home/Free.js [hunk 1]
```diff
@@ -15,21 +15,52 @@ function Free() {
...
-        <h1>Free. &nbsp;For anyone and everyone.</h1>
+        <h1>Free. &nbsp;For everyone.</h1>
         <p>
-          Node Warrior is developed by a group of passionate software
-          engineers who believe that quality computer science resources
-          should be available to all.
+          Destructure.io is a platform created by software engineers who believe quality programming resources
+          should be accessible to anyone, anywhere.
         </p>
+        <div style={{ "display": "inline-block" }}>
```
Comment (ConnerArdman) [inline src/components/home/Free.js]: We should probably do this in CSS but eh its late

File: src/components/home/HomePage.js [hunk 2]
```diff
@@ -26,20 +25,7 @@ function HomePage(props) {
         <div id="free-container" className="content-container">
           <Free />
         </div>
-        <div className="team-container">
-          <Team />
```
Comment (ConnerArdman) [inline src/components/home/HomePage.js]: You can delete team.js also if you want

Comment (hawkticehurst) [reply src/components/home/HomePage.js]: Nah, I intentionally didn't delete it, so we can potentially use it as a reference for later for the About page.

File: src/components/home/Free.js (hunk 1 above)
Comment (hawkticehurst) [reply src/components/home/Free.js]: I thought about that too, but that's literally the only style this div needs and I couldn't think of a good class name for it 😅

Comment (ConnerArdman) [reply src/components/home/Free.js]: idk you could literally just call it inline-block if you really want

Comment (hawkticehurst) [reply src/components/home/Free.js]: Lol actually yeah I should because I remembered I do this elsewhere too.

**Pull request merged**